test/ValidationFramework/data/gmat_astrodynamics/gmat_scripts/scenario004-mission-sequence.script

```diff
@@ -0,0 +1,230 @@
+% Copyright Loft Orbital Solutions Inc.
```

Comment:
Just to be 100% sure: This is what is basically built by the gmat tooling to then run and obtain the output csv: I don't think we need them here (totally agree we should keep them in the artifacts though 💯 !)